shuffler/auxiliary/NW Sea/Bannan Island/Bannan.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../../../aux_schema.json",
+  "name": "Bannan",
+  "rooms": [
+    {
+      "name": "OutsideLeft",
+      "chests": [
+        {
+          "name": "DigSpot",
+          "type": "dig_spot",
+          "contents": "treasure_map_SW_5",
+          "zmb_actor_index": 4,
+          "zmb_file_path": "Map/isle_hidari/map00.bin/zmb/isle_hidari_00.zmb"
+        },
+        {
+          "name": "IslandChest",
+          "type": "chest",
+          "contents": "power_gem",
+          "zmb_file_path": "Map/isle_hidari/map00.bin/zmb/isle_hidari_00.zmb",
+          "zmb_mapobject_index": 111
+        }
+      ],
+      "doors": [
+        {
+          "name": "WayfarerHouse",
+          "link": "WayfarerHouse.Main.Entrance"
+        },
+        {
+          "name": "BannanGrottoEntrance",
+          "link": "BannanGrotto.LeftSide.OutsideLeft"
+        }
+      ]
+    },
+    {
+      "name": "OutsideRight",
+      "chests": [
+        {
+          "name": "CannonMiniGameBombBag",
+          "type": "npc",
+          "contents": "todo",
+          "bmg_file_path": "todo",
+          "bmg_instruction_index": -1
+        },
```

Comment:
This doesn't seem to be located in the ZMB actors as I would expect. It might be yet another custom item type, I opened an issue #173 and will leave this as "TODO" for now